test-utils/src/main/groovy/net/neoforged/gradle/utils/test/extensions/RuntimeBuilderExtensions.groovy

```diff
@@ -0,0 +1,23 @@
+package net.neoforged.gradle.utils.test.extensions
+
+import net.neoforged.gradle.common.caching.CentralCacheService
+import net.neoforged.trainingwheels.gradle.functional.builder.Runtime
+
+/**
+ * Extensions for the [Runtime.Builder] class.
+ */
+class RuntimeBuilderExtensions {
```

Comment:
Missing compilestatic

Reply:
Can't be compile static, causes compiler errors in consumers.